Add function for quickly searching the last message of a user in an introductions channel

Delta-gabged opened the pull request:
PR includes changes in the `Resources.json` that correspond to the LLCC server and introductions channel (and how it'd be structured in the Yaml file)

Introductions channel:

![testcase-3](https://user-images.githubusercontent.com/84141734/118355067-b15d2b00-b5a0-11eb-894d-4b0b47bf2204.PNG)

General channel (using the feature):

![testcase-1](https://user-images.githubusercontent.com/84141734/118355083-c76aeb80-b5a0-11eb-9fad-f7374f63b5b7.PNG)

![testcase-2](https://user-images.githubusercontent.com/84141734/118355087-cb970900-b5a0-11eb-8c18-7822ec92e9bc.PNG)

![testcase-4](https://user-images.githubusercontent.com/84141734/118355089-cdf96300-b5a0-11eb-9341-5a1ed804384b.PNG)

Current questions:

* Will the simple for-loop be fine for the use case (or if there should be a better way to go about it)
* Will a simple substring search be fine (or if I should do something more specific like prioritizing display name or using fuzzy search)
* Will I have to suppress link previews

JacobMintzer commented on cogs/guildFunctions.py:
```diff
@@ -392,6 +392,51 @@ async def seiyuu(self, ctx, *, role=None):
+				for _u in _user_matches:
+					output_msg += "\t{} ({})\n".format(_u.display_name, _u.name)
+				await ctx.send(output_msg)
+			elif len(_user_matches) == 0:
```
You should return from the function when you don't get exactly one to prevent checking if every message in the intro channel's author is none

JacobMintzer commented on cogs/guildFunctions.py:
```diff
@@ -392,6 +392,51 @@ async def seiyuu(self, ctx, *, role=None):
+				user = _user_matches[0]
+
+		_chid = self.bot.config["introChannel"][_guild]["channel"]
+		_ch = self.bot.get_channel(_chid)
```
This could be fairly time consuming, in order to let the user know that Haruka got the message and is processing it, it's good to wrap either this or the entire function in a `async with ctx.typing()` block

Delta-gabged commented:
* Addressed comments
* Added link suppression

I wasn't sure if there was a way to suppress links before sending them (the solution I found was to remove the embed after sending the message, which still showed it for a split second), so I instead found links via regex and surrounded them with `<>`.

![image](https://user-images.githubusercontent.com/84141734/119527577-083dde00-bdb3-11eb-9383-a52c9bbdbf58.png)

JacobMintzer commented:
LGTM, feel free to merge it if you are satisfied with how it is. Not sure how fast it'll run in prod, but there's only one way to find out!
